src/components/BulletCursorOverlay.tsx

```diff
@@ -0,0 +1,135 @@
+import { useRef } from 'react'
+import { useSelector } from 'react-redux'
+import { css } from '../../styled-system/css'
+import Path from '../@types/Path'
+import SimplePath from '../@types/SimplePath'
+import useBulletPosition from '../hooks/useBulletPosition'
+import attributeEquals from '../selectors/attributeEquals'
+import isContextViewActive from '../selectors/isContextViewActive'
+import rootedParentOf from '../selectors/rootedParentOf'
+import head from '../util/head'
+import parentOf from '../util/parentOf'
+
+/** A larger circle that surrounds the bullet of the cursor thought. */
+const BulletCursorOverlay = ({
+  isCursorActive,
+  x = 0,
+  y = 0,
+  simplePath,
+  path,
+}: {
+  isCursorActive?: boolean
```

Comment:
Is there a reason to pass `isCursorActive` as a prop if it is just used for a conditional render?

If there is additional complexity in the condition I sometimes push it down to the child component, otherwise I usually control rendering in the parent component.

Reply:
Yep, makes sense. I'm gonna move the conditional logic up to the LayoutTree level instead